doc/releases/release-notes-2.1.rst

```diff
@@ -46,7 +46,46 @@ Bluetooth
 Build and Infrastructure
 ************************
 
-* TBD
+* Various parts of the binding format have been simplified. The format is
+  better documented now too.
+
+  This turns e.g.
+
+  .. code-block:: yaml
+
+     parent:
+         bus: spi
+
+     inherits:
+         !include foo.yaml
+
+     properties:
+         compatible:
+             constraint: "company,device"
+             type: string-array
+
+         frequency:
+             type: int
+             category: required
+
+  into
```

Comment:
``you should write this simplified description instead:``

Reply:
Rephrased now.